mockserver/__init__.py

```diff
@@ -8,8 +8,8 @@ def __init__(self, base_url):
         self.base_url = base_url
         self.expectations = []
 
-    def _call(self, command, data=None):
-        return requests.put("{}/{}".format(self.base_url, command), data=data)
+    def _call(self, command, data=None, **kwargs):
```

Comment:
i personnally don't enjoy kwargs in code we control, i think it obfuscates usage, it's a pain to debug what's coming in and out, and screws auto completion / parameters inferences